examples/getstarted/src/api/kitchensink/content-types/kitchensink/schema.json

```diff
@@ -156,6 +156,13 @@
       "type": "customField",
       "regex": "^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$",
       "customField": "plugin::color-picker.color"
+    },
```

Comment:
did you mean to commit changes to the example files?

Reply:
I think we should, it's getting tiresome making the same component over and over again every time we look at this area 🥲